src/dao/api.dao.ts

```diff
@@ -105,6 +105,18 @@ export class ClaimDao {
       OR: [
         { subject: { contains: search, mode: "insensitive" } },
         { object: { contains: search, mode: "insensitive" } },
+        { claim: { contains: search, mode: "insensitive" } },
+        { statement: { contains: search, mode: "insensitive" } },
+        { sourceURI: { contains: search, mode: "insensitive" } },
```

Comment:
i think it might be slow if we include all these, i like including the claim and statement but please remove the rest following this for now

its also important not to return spurious results too

Reply:
that's fine, I will just leave the statement